Makefile

```diff
@@ -67,6 +67,11 @@ build-push-all: container-build-operator container-push-operator container-build
 
 upgrade-test:
 	./hack/upgrade-test.sh
+
+dump-state:
+	CMD="./cluster-up/kubectl.sh" ./hack/dump-state.sh error
```

Comment:
👍 very useful script.  I think this script could be helpful outside of cluster-up, so we may want to avoid setting `CMD` here and have `upgrade-test.sh` set it.

Here's a place where it could also be used: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/hack/deploy.sh#L118-L125

Reply:
I tried this before in https://github.com/kubevirt/hyperconverged-cluster-operator/pull/314/commits but  hack/common.sh is setting up the wrong value for CMD (at least for the environment that does the CI testing in this environment)

Reply:
ok we can table that addition for now.  Would like to come back to it later though